### DISCUSSION TITLE
Shortcut to specific  item

### DISCUSSION THREAD

**Discussion by boyanmolina** [hluk/CopyQ#2180] [Ideas]
How do you make a command that will be triggered by a global shortcut and will paste a specific item? (without opening the program)

**Comment by hluk** [comment]
There are some commands that allow the items to be pasted with a global shortcut. See: https://github.com/hluk/copyq-commands/tree/master/Global#select-nth-item (see [README.md](https://github.com/hluk/copyq-commands/blob/master/README.md) for details on how to add the command).
